indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -21,13 +21,14 @@
     [cmr.elastic-utils.connect :as es-util]
     [cmr.indexer.config :as config]
     [cmr.indexer.data.concept-parser :as cp]
+    [cmr.indexer.data.concepts.deleted-granule :as dg]
     [cmr.indexer.data.elasticsearch :as es]
     [cmr.indexer.data.humanizer-fetcher :as humanizer-fetcher]
     [cmr.indexer.data.index-set :as idx-set]
     [cmr.indexer.data.metrics-fetcher :as metrics-fetcher]
     [cmr.message-queue.config :as qcfg]
-    [cmr.message-queue.services.queue :as queue]
     [cmr.message-queue.queue.queue-protocol :as queue-protocol]
+    [cmr.message-queue.services.queue :as queue]
```

Comment:
Remove this line. I don't think this is used anywhere.

Reply:
```
(defn publish-provider-event
  "Put a provider event on the message queue."
  [context msg]
  (let [queue-broker (get-in context [:system :queue-broker])
        exchange-name (config/provider-exchange-name)]
    (queue/publish-message queue-broker exchange-name msg)))
```

Reply:
oh, missed that. Then keep it. :)